src/fiatExchanges/ExternalExchanges.test.tsx

```diff
@@ -23,58 +21,18 @@ const mockStore = createMockStore({
   },
 })
 
-const MOCK_EXCHANGES_FETCH_RESPONSE: ExternalExchangeProvider[] = [
-  {
-    name: 'Bittrex',
-    link: 'https://bittrex.com/Market/Index?MarketName=USD-CELO',
-    currencies: [Currency.Celo, Currency.Dollar],
-    supportedRegions: ['global'],
-  },
-  {
-    name: 'CoinList Pro',
-    link: 'https://coinlist.co/asset/celo',
-    currencies: [Currency.Celo, Currency.Dollar],
-    supportedRegions: ['global'],
-  },
-  {
-    name: 'OKCoin',
-    link: 'https://www.okcoin.com/en/spot/trade/cusd-usd/',
-    currencies: [Currency.Celo, Currency.Dollar],
-    supportedRegions: ['global'],
-  },
-]
-
+// Removed mockFetch testing as fetchExchanges is called in SelectProvider
```

Comment:
This comment is unnecessary. Its referencing code that will cease to exist as soon as the PR is merged